Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.6-slim
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+RUN apt-get install -y vim-tiny krb5-user krb5-config libkrb5-dev \
```

Comment:
It would be good to put one package per line, and sort packages by name, as in r-w-e-yadage.  This is nicer WRT future diffs as we'll be adding/removing packages from the lenghy `apt-get` command.

Example:

```
RUN apt-get update && \
    apt-get install -y \
      autoconf \
      automake \
      gcc \
      graphviz \
      graphviz-dev \
      imagemagick \
      libffi-dev \
      libtool \
      openssl \
      python-dev \
      python-pip \
      unzip \
      vim-tiny
```

Also, we could terminate by removing unneeded files to make the image a bit slimmer:

```
...
     vim-tiny && \
   apt-get -y autoremove && \
   apt-get -y clean
```